.vscode/settings.json

```diff
@@ -1,3 +1,7 @@
 {
-  "prettier.requireConfig": true
-}
+    "standard.validate": [
```

Comment:
I'm thinking we should probably remove this from being tracked in git unless we want to force folks to use the same linting extensions

Reply:
Yeah let's add the whole .vscode folder to the gitignore. I think we should be using the same linting rules but we don't need to impose limitations on what extensions are used to highlight + fix them (unless we find that one works best for some reason)